src/sdk/object_io.js

```diff
@@ -495,6 +496,7 @@ class ObjectIO {
             });
             await mc.run();
             if (mc.had_errors) throw new Error('Upload map errors');
+            if (params.update_cache_stats_hook) params.update_cache_stats_hook(params.range.end - params.range.start);
```

Comment:
set the name to `upload_chunks_hook`:
```suggestion
            if (params.upload_chunks_hook) params.upload_chunks_hook(params.range.end - params.range.start);
```